src/content/docs/apis/nerdgraph/examples/mobile-monitoring-config-nerdgraph.mdx

```diff
@@ -0,0 +1,100 @@
+---
+title: "NerdGraph tutorial: Mobile agent monitoring examples"
+tags:
+  - APIs
+  - NerdGraph
+  - Examples
+  - Browser
+metaDescription: Examples of using New Relic NerdGraph API to adjust a mobile agent
+redirects:
+  - /docs/apis/nerdgraph/examples/export-dashboards-pdfpng-api-tutorial
+  - /docs/apis/nerdgraph/examples
+---
+
+You can create mobile applications in the NerdGraph API instead of using the UI. The advantage to this is that you can programmatically create and retrieve the application token to copy and paste into your mobile application.
```

Comment:
```suggestion
You can create mobile applications in the NerdGraph API instead of using the UI. The advantage to this is that when it's time to instrument your mobile application with New Relic, you can programmatically create and retrieve the application token to copy and paste into your mobile application.
```